codespell_lib/_codespell.py

```diff
@@ -28,6 +28,11 @@
 import textwrap
 
 word_regex_def = u"[\\w\\-'’`]+"
+# While we want to treat characters like ( or " as okay for a starting break,
+# these may occur unescaped in URIs, and so we are more restrictive on the
+# endpoint.  Emails are more restrictive, so the endpoint remains flexible.
```

Comment:
What about `mailto:someone@foo.com?subject=Test`?

Reply:
Added a test. It's caught with the current regex due to \b matching both : and ?